### ISSUE TITLE
bower install failure

### ISSUE THREAD

**Issue by Zoltu** [systemjs/systemjs#630]
1. `git clone git@github.com:systemjs/systemjs.git`
1. `cd systemjs`
1. `npm install`
1. `npm install bower`
1. `node_modules/.bin/bower install`
```
? May bower anonymously report usage statistics to improve the tool over time? Yes
bower                       EMALFORMED Failed to read C:\Users\Micah\Source\systemjs\bower.json

Additional error details:
Unexpected string
```

I glanced at the bower.json file and everything looks fine.  I'm not sure how to go about debugging this further.  :/

Windows 8.1
Node 0.12.2

**Comment by guybedford** [comment]
It's a missing comma in the bower.json file - https://github.com/systemjs/systemjs/blob/master/bower.json#L9.

**Comment by Zoltu** [comment]
Fixed in 767f8134fa7bd9add27ca086d5ea633364567475
